process/errors.go

```diff
@@ -408,3 +405,9 @@ var ErrUnknownBlockType = errors.New("block type is unknown")
 
 // ErrMissingPreProcessor signals that required pre processor is missing
 var ErrMissingPreProcessor = errors.New("pre processor is missing")
+
+// ErrNilTxValidator signals that a nil tx handler validator has been provided
+var ErrNilTxValidator = errors.New("nil tx handler validator provided")
```

Comment:
The name and comment of this error is confusing because of the nilTxValidator struct. Does it mean that the tx validator is nil, or the tx validator is nilTxValidator? Same for the next error.